src/index.html

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <html lang="en">
-  <head>
+  <head class="header">
```

Comment:
The `class` attribute is not appropriate for the `<head>` element as this element is not meant to be styled. Please remove the `class` attribute.